orderbook/src/event_updater.rs

```diff
@@ -113,7 +125,7 @@ impl EventUpdater {
             .all_events()
             .from_block((*block_range.start()).into())
             .to_block((*block_range.end()).into())
-            .block_page_size(1000)
+            .block_page_size(500)
```

Comment:
Decreased this since Felix mentioned our event updating might be loading nodes a lot